src/main/java/io/anserini/search/SearchTweets.java

```diff
@@ -130,24 +132,24 @@ public static void main(String[] args) throws Exception {
     for ( MicroblogTopic topic : topics ) {
       long curQueryTime = System.nanoTime();
```

Comment:
I know you didn't change this line, but given that you're converting to ms later on, would `system.currentTimeMillis()` not be the better call to make? Also, `curQueryTime` isn't a super amazing name for this. It's really `startQueryTime` or even maybe just `start`?